heron/spi/src/java/BUILD

```diff
@@ -171,8 +171,7 @@ java_library(
 java_binary(
     name = "spi-unshaded",
     srcs = glob([
-        "**/spi/utils/**/*.java",
-        "**/spi/common/**/*.java",
+        "**/spi/**/*.java",
```

Comment:
Instead of specifying everything under spi - I would suggest module by module - helps to keep track of it in a cleaner fashion.